index.html

```diff
@@ -0,0 +1,129 @@
+        <div class="flex-h">
+          <a class="flex-h align-center m-r-2" href=""
+            ><img src="./images/mint.png" alt="mint" /><span>mint</span></a
+          >
+          <a class="flex-h align-center m-r-2" href=""
+            ><img src="./images/quickbooks.png" alt="quickbooks" /><span
+              >quickbooks</span
+            ></a
+          >
+          <a class="flex-h align-center m-r-2" href=""
+            ><img src="./images/turbotax.png" alt="turbotax" /><span
+              >turbotax</span
+            ></a
+          >
+        </div>
+      </footer>
+    </div>
+    <!--End of Logo Section-->
+
+    <main id="signup-form" class="flex-v align-center m-t-1">
+      <form action="#" class="p-2">
+        <header class="flex-v align-center m-b-1">
+          <span class="m-b-1">Create an Intuit account</span>
+          <p>One account for everything Intuit, including Mint.</p>
+          <a href="">Learn more</a>
+        </header>
+
+        <div class="form-input">
+          <label for="email">Email address</label
+          ><input type="email" name="email" id="email" />
+        </div>
+
+        <div class="form-input">
+          <label for="tel">Phone (recommended)</label
+          ><input type="tel" name="tel" id="tel" />
+          <span>Standard call, messaging or data rates may apply.</span>
+        </div>
+
+        <div class="form-input" id="pass">
+          <label for="password">Password</label
+          ><input type="password" name="password" id="password" />
+          <i class="fas fa-unlock-alt"></i>
+        </div>
+
+        <div class="form-input">
+          <button class="btn">
+            <i class="fas fa-lock"></i> Create Account
+          </button>
+        </div>
+
+        <div id="terms">
+          <em>
+            By clicking Create account, you agree to our<br />
+            <a href="#">Terms</a> and have read and acknowledge our
+            <a href="#">Privacy <br />Statement</a>.
+          </em>
+        </div>
+      </form>
+      <footer class="flex-h align-center">
+        <p>
+          Invisible reCAPTCHA by Google <a href="#">Privacy Policy</a> and
+          <a href="#">Terms of Use</a>.
+        </p>
+      </footer>
+    </main>
+    <!-- End of Main section -->
+
+    <footer id="main-footer" class="flex-v align-center m-t-1">
+      <div>
+        <a href="">Legal</a>
+        <a href="">Privacy</a>
+        <a href="">Security</a>
+      </div>
+
+      <div id="copyright" class="m-b-1">
+        <p>
+          © 2020 Intuit, Inc. All rights reserved. Intuit, QuickBooks, QB,
+          TurboTax, ProConnect and Mint are registered trademarks of Intuit Inc.
+        </p>
+
+        <p>
+          Terms and conditions, features, support, pricing, and service options
+          subject to change without notice.
+        </p>
+      </div>
+
+      <div id="footer-privacy" class="flex-h">
+        <div class="btn">Privacy Settings</div>
+      </div>
```

Comment:
Remove this section called `Privacy Settings` in your project as it doesn't exist on the original webpage👆
Refer to screenshot👇
![ice_screenshot_20200331-115351](https://user-images.githubusercontent.com/33323077/78019633-f0ea9880-7347-11ea-91e2-3abe0ad31cab.png)